introduction.md

```diff
@@ -1,3 +1,8 @@
 Hi, I am Peter and I am learning Mastering GitHub on CodeSchool. :)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?